README.md

```diff
@@ -23,8 +23,8 @@ like finding and fixing bugs and improving our documentation or translations!
 
 To get started you can try [GraphHopper Maps](README.md#graphhopper-maps), read through our documentation and install the GraphHopper Web Service locally.
 
- * stable 2.0:   [documentation](https://github.com/graphhopper/graphhopper/blob/2.0/docs/index.md), [web service jar](https://graphhopper.com/public/releases/graphhopper-web-2.0.jar), [announcement](https://www.graphhopper.com/blog/2020/09/30/graphhopper-routing-engine-2-0-released/)
- * unstable 3.0: [documentation](https://github.com/graphhopper/graphhopper/blob/master/docs/index.md), [web service jar](https://oss.sonatype.org/content/groups/public/com/graphhopper/graphhopper-web/2.0-SNAPSHOT/)
+ * stable 2.x: [documentation](https://github.com/graphhopper/graphhopper/blob/2.x/docs/index.md), [web service jar](https://graphhopper.com/public/releases/graphhopper-web-2.0.jar), [announcement](https://www.graphhopper.com/blog/2020/09/30/graphhopper-routing-engine-2-0-released/)
+ * unstable 3.x: [documentation](https://github.com/graphhopper/graphhopper/blob/master/docs/index.md), [web service jar](https://oss.sonatype.org/content/groups/public/com/graphhopper/graphhopper-web/2.0-SNAPSHOT/)
```

Comment:
should it not be 3.0-SNAPSHOT here?

Reply:
hm and maybe remove the web service jar for 3.x altogether?

Reply:
Yes, will remove it

Reply:
And maybe change `unstable 3.x` to `unstable master`? The link links to master and there is no 3.x branch anyway?

Reply:
Uh, and something is broken about the formatting: 
![image](https://user-images.githubusercontent.com/17603532/95335336-8fcfee80-08af-11eb-92b9-6fec5d3c4fb0.png)

Reply:
:D .. ups. Thanks. Will try to fix it!

Reply:
Fixed. An separate line seems to be important: https://github.com/graphhopper/graphhopper/commit/7dafe4b84924c43ba0ad9d8a8220e73133583159

Reply:
Thanks! Looks good :)